spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -130,6 +130,10 @@ def export_compliance_missing?
       external_state == BUILD_STATES[:export_compliance_missing]
     end
 
+    def self.processed?
```

Comment:
Thanks so much for bringing this to our attention! I can't believe with all the unit tests we added that we missed this...

Anyhow, here is a PR to fix it: https://github.com/fastlane/fastlane/pull/9025
As soon as we can get a 🚢 on it, we will release a new build.